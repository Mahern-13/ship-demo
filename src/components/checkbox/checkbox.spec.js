import React from "react";
import Checkbox from "./index";
import { mount, shallow } from "enzyme";
import { Provider } from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore();
const store = mockStore();

describe("Checkbox", () => {
  it("renders the component as checked", () => {
    const wrapper = mount(
      <Provider store={store}>
        <Checkbox
          label="text"
          stop={{
            address: "address",
            completed: true,
            id: "ef457741-fb6d-4650-83c3-b786b099fed8",
            name: "name",
            recommendedAddress: "111 address",
            verified: true
          }}
        />
      </Provider>
    );

    expect(wrapper.exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
