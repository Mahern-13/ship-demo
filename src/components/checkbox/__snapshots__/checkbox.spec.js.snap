// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox renders the component as checked 1`] = `ReactWrapper {}`;
